src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,45 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+/**
+ * Represents the command to mark specific Tasks in TaskList as completed or not completed
+ * that inherits from Command.
+ */
+public class MarkCommand extends Command {
+    private final String userInput;
+    private final boolean isMark;
```

Comment:
Maybe you consider changing the name of the class to better represent both marking and unmarking command.